bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -442,6 +443,98 @@ impl Height {
     /// assert_eq!(lock_time.to_consensus_u32(), n_lock_time);
     #[inline]
     pub fn to_consensus_u32(self) -> u32 { self.0 }
+
+    /// Increments height by 1, saturating at `Height::MAX`.
+    #[inline]
+    pub const fn increment(self) -> Height { self.saturating_add(1) }
+
+    /// Decrements height by 1, saturating at `Height::MIN`.
+    #[inline]
+    pub const fn decrement(self) -> Height { self.saturating_sub(1) }
```

Comment:
These seem arbitrary. There's a reason Rust doesn't have the `++` operator. I suggest having a method called `offset` that just accepts an int.